### PR TITLE
Add paperclip

### PR DESCRIPTION
**Pull request by sd-kin**

### CODE REVIEW

File: app/models/concerns/imageable.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Imageable
+  extend ActiveSupport::Concern
+
+  included do
+    has_many :images, as: :imageable, dependent: :destroy
```
Comment (kirushik) [inline app/models/concerns/imageable.rb]: Will actual files be destroyed here, or only their database handles?

File: config/routes.rb [hunk 2]
```diff
@@ -66,8 +66,9 @@
 
   resources :polls, concerns: :statusable do
     member do
-      post 'choose'
-      get  'result'
+      post   'choose'
+      get    'result'
+      delete 'delete_image'
```
Comment (kirushik) [inline config/routes.rb]: I didn't see this action in `PollsController`. Am I missing something?

**Review by kirushik — CHANGES_REQUESTED** [review]
I don't see any integration test for images attaching.
Since it's a pretty advanced (and fragile) thing, with lots of functionality "hidden" in gems — I think it will make sense to test image uploading with PhantomJS.

What do you think?

File: app/models/concerns/imageable.rb (hunk 1 above)
Comment (sd-kin) [inline app/models/concerns/imageable.rb]: Paperclip take care about actual files, so both actual files and database handles will be deleted. I doubt it myself, but it just works.

File: config/routes.rb (hunk 2 above)
Comment (sd-kin) [inline config/routes.rb]: nope, I forgot to glean up.

File: config/environments/test.rb [hunk 3]
```diff
@@ -41,11 +41,9 @@
   # config.action_view.raise_on_missing_translations = true
 
   # paperclip file store location
-  Paperclip::Attachment.default_options[:path] = "#{Rails.root}/spec/test_files/:class/:id_partition/:style.:extension"
+  Paperclip::Attachment.default_options[:path] = ':rails_root/public:url'
+  Paperclip::Attachment.default_options[:url]  = '/test_files/:class/:id_partition/:style/:filename'
```
Comment (kirushik) [inline config/environments/test.rb]: Why do you think this is better?
I can't find anyone messing with their `:url` option of Paperclip for testing purposes (see [here](https://github.com/search?utf8=%E2%9C%93&q=Paperclip%3A%3AAttachment.default_options%5B%3Aurl%5D+filename%3Atest.rb&type=Code&ref=searchresults))

**Review by kirushik — APPROVED** [review]

**Pull request merged**